Library/Formula/mlkit.rb

```diff
@@ -0,0 +1,29 @@
+class Mlkit < Formula
+  desc "Compiler for the Standard ML programming language"
+  homepage "https://melsman.github.io/mlkit"
+  url "https://github.com/melsman/mlkit/archive/mlkit-4.3.8.tar.gz"
+  sha256 "337ea3d333c694871f05f9c3ecbf1c05d4bcd1a0993e87801507930a1637a5d4"
+
+  bottle do
+    sha256 "32bcf016846a33d8794230567d6b41ab7d703a13eff4568b318073ba2fe61426" => :el_capitan
+  end
+
+  depends_on "autoconf" => :build
+  depends_on "mlton" => :build
+  depends_on :tex
```

Comment:
We are using it for two purposes:
1. documentation for some enclosed tools (built from source)
2. generation of test reports

For point (1), I can commit generated pdfs instead of building the pdfs
from the tex sources at every build.

For point (2), we should really generate html instead...

Den fredag den 11. marts 2016 skrev Dominyk Tiller <notifications@github.com
>:

> In Library/Formula/mlkit.rb
> <https://github.com/Homebrew/homebrew/pull/49933#discussion_r55850097>:
>
> > @@ -0,0 +1,29 @@
> > +class Mlkit < Formula
> > +  desc "Compiler for the Standard ML programming language"
> > +  homepage "https://melsman.github.io/mlkit"
> > +  url "https://github.com/melsman/mlkit/archive/mlkit-4.3.8.tar.gz"
> > +  sha256 "337ea3d333c694871f05f9c3ecbf1c05d4bcd1a0993e87801507930a1637a5d4"
> > +
> > +  bottle do
> > +    sha256 "32bcf016846a33d8794230567d6b41ab7d703a13eff4568b318073ba2fe61426" => :el_capitan
> > +  end
> > +
> > +  depends_on "autoconf" => :build
> > +  depends_on "mlton" => :build
> > +  depends_on :tex
>
> Sure, if you're happy with that. Are you using it just for documentation
> or functionality beyond that?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/49933/files#r55850097>.
>